README.md

```diff
@@ -11,9 +11,21 @@ ScalarDB is a universal HTAP engine. It achieves ACID transactions and real-time
   <a href='https://www.youtube.com/watch?v=lW6nCEJV-gs' target="_blank"><img src='docs/images/scalardb-video-preview.png' width='70%'></a>
 </p>
 
-## Docs
+To learn more, see [ScalarDB Overview](https://scalardb.scalar-labs.com/docs/latest/overview).
 
-See our [User Documentation](https://scalardb.scalar-labs.com/docs/latest/).
+## Quickstart
+
+1. Start running transactions.
+   - **Get started with ScalarDB Core:** To set up a basic application by using Java, see [Get Started with ScalarDB](https://scalardb.scalar-labs.com/docs/latest/getting-started-with-scalardb).
```

Comment:
```suggestion
   - **Get started with ScalarDB Core:** To set up a basic application by using Java, see [Getting Started with ScalarDB](https://scalardb.scalar-labs.com/docs/latest/getting-started-with-scalardb).
```
The title of the linked document is a bit different. Is it better to use the same name as the linked document title?

Reply:
In the future, I think we should change the titles to start with a verb since the document is a task (action for the user to do). For now, let's use the current title here and update it later.

Reply:
I see. I understood the future plan! Thank you!